x-pack/plugins/canvas/server/sample_data/load_sample_data.js

```diff
@@ -20,13 +21,33 @@ export function loadSampleData(server) {
     });
   }
 
+  const sampleDataLinkLabel = i18n.translate('xpack.canvas.sampleDataLinkLabel', {
+    defaultMessage: 'Canvas',
+  });
   server.addSavedObjectsToSampleDataset(
     'ecommerce',
     updateCanvasWorkpadTimestamps(ecommerceSavedObjects)
   );
+  server.addAppLinksToSampleDataset('ecommerce', {
+    path: '/app/canvas#/workpad/workpad-e08b9bdb-ec14-4339-94c4-063bddfd610e',
+    label: sampleDataLinkLabel,
+    icon: 'canvasApp',
+  });
+
   server.addSavedObjectsToSampleDataset(
     'flights',
     updateCanvasWorkpadTimestamps(flightsSavedObjects)
   );
+  server.addAppLinksToSampleDataset('flights', {
+    path: '/app/canvas#/workpad/workpad-a474e74b-aedc-47c3-894a-db77e62c41e0',
+    label: sampleDataLinkLabel,
+    icon: 'canvasApp',
+  });
+
   server.addSavedObjectsToSampleDataset('logs', updateCanvasWorkpadTimestamps(webLogsSavedObjects));
+  server.addAppLinksToSampleDataset('logs', {
+    path: '/app/canvas#/workpad/workpad-5563cc40-5760-4afe-bf33-9da72fac53b7',
+    label: sampleDataLinkLabel,
+    icon: 'canvasApp',
+  });
```

Comment:
minor, but there's a slight bit of duplicated literal strings in this file. Maybe add some `consts` at the top of the file?

Reply:
I will leave that up to @w33ble

Reply:
I'm find with it, but if you wanted de-dupe stuff a bit, you could create a little helper function:

```js
const getAppLinkSettings = (workpadId, icon = 'canvasApp') => ({
  path: `/app/canvas#/workpad/${workpadId}`,
  label: sampleDataLinkLabel,
  icon: icon,
});
```

Then you can use that: 

```js
server.addAppLinksToSampleDataset('ecommerce', getAppLinkSettings('workpad-e08b9bdb-ec14-4339-94c4-063bddfd610e'));
```

But I think it's totally fine the way it is too.